src/object/cli_mod.c

```diff
@@ -186,7 +186,7 @@ dc_obj_init(void)
 	if (rc != 0)
 		goto out_class;
 
-	obj_coll_thd = OBJ_COLL_THD_MIN;
+	obj_coll_thd = 0; /* Was OBJ_COLL_THD_MIN, restore when leak is fixed */
```

Comment:
minor nit: I really prefer to put some kind of tracker for these kinds of TODO/FIXME changes. e.g. `/* FIXME b/360833010: Restore to OBJ_COLL_THD_MIN when leak fixed */`

Even better, use a new bug that can be closed when the FIXME is reverted. We do that a lot in the control plane code. Makes it harder to forget when there's an open ticket that gets looked at during planning.